File: 01-Assignment-02-02-2026/Using_C/program_32.c
/* Program 32: Print Fibonacci series using iteration */

#include <stdio.h>

int main() {
    int n;
    int a = 0, b = 1, next;

    printf("Enter number of terms: ");
    scanf("%d", &n);

    if (n <= 0) {
        printf("Invalid number of terms\n");
        return 0;
    }

    printf("Fibonacci Series: ");

    for (int i = 1; i <= n; i++) {
        printf("%d ", a);
        next = a + b;
        a = b;
        b = next;
    }

    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program32.java
/*
Program 32
Java program to print Fibonacci series using iteration
*/

import java.util.Scanner;

public class Program32 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter number of terms: ");
        int num = scanner.nextInt();

        if (num < 0) {
            System.out.println("Invalid input");
        } else {
            int a = 0, b = 1;

            for (int i = 0; i < num; i++) {
                System.out.print(a + " ");
                int next = a + b;
                a = b;
                b = next;
            }
        }
        scanner.close();
    }
}